v0/destinations/delighted/util.js

```diff
@@ -0,0 +1,94 @@
+
+const axios = require("axios");
+const {
+    CustomError,
+    getValueFromMessage
+} = require("../../util");
+const { ENDPOINT } = require("./config");
+
+
+function isValidEmail(email){
```

Comment:
how about using es6 arrow function for `isValidEmail` and `isValidPhone` as well?